dio/lib/src/options.dart

```diff
@@ -41,6 +41,10 @@ enum ResponseType {
 
   /// Get the original bytes, the [Response.data] will be [List<int>].
   bytes,
+
+  /// Get blob url.
+  /// Only work in web.
```

Comment:
The description seems to lack explanation.

Reply:
I made changes to the description